install_template/templates/products/migration-toolkit/index.njk

```diff
@@ -24,14 +24,14 @@ legacyRedirects:
 {% endblock frontmatter %}
 {% block navigation %}
 {{- super() -}}
-- install_on_mac
-- install_on_windows
+- macos
+- windows
 - installing_jdbc_driver
 {% endblock navigation %}
 
 {% block otherosinstall %}
-## Macintosh
-- [Mac OS X](macos)
+## Mac
+- [macOS](macos)
```

Comment:
We are no longer building macOS installers. References to macOS should be removed.